### DISCUSSION TITLE
How to uninstall it

### DISCUSSION THREAD

**Discussion by gtresnandika** [Z3d0X/filament-logger#100] [Q&A]
hi,thanks for nice plugins. Its a great plugins for records users activity in the app.
but i have to uninstall it since i have 40K users that sometimes make mysql process is overloaded. sadly i need to uninstall this plugins. can you guide me how to do it properly?

**Comment by Z3d0X** [comment]
just like any other composer package?
`composer remove z3d0x/filament-logger`.

Additionally, If you wish to erase logs already created, you may drop the `activity_log` table

**Comment by gtresnandika** [comment]
thank you so much

**Comment by gtresnandika** [comment]
oh forgot..im using filament v3,do i need to change appprovider too?
